### ISSUE TITLE
Unable to translate image files in Korean for no l10n (ko) space.

### ISSUE THREAD

**Issue by jmyung** [kubernetes/website#15794]
## This is a Feature Request
I am a korean localization member of kubernetes/kubernetes-docs-ko channel, and now I am working on a [service](https://kubernetes.io/docs/concepts/services-networking/service/) page.
I have an issue with translation of image files, because I can't find the way how to translate image file in Korean.

> **There is no image space for `ko`.**

## What would you like to be added
It seems that there are two type of image files.

**1. SVG and PNG files**
Please checkout below link.
https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-userspace
I guess that this image comes from static\images\docs in website repo.
  * static\images\docs\services-userspace-overview.svg
  * static\images\docs\services-userspace-overview.png

and I would like to translate like this.
![image](https://user-images.githubusercontent.com/25141106/62824102-30574f80-bbd4-11e9-8976-f739590b010a.png)

However, there is no space for korean and we don't have authorization to submit PR that location.

**2. image file only (e.g. PNG)**
- static\images\docs\ui-dashboard.png

There is no space for korean and we don't have authorization to submit PR that location.

## Why is this needed
In conclusion, There is `no space` for korean image, when we try to convert to korean.

## Comments
I think that this might be same issue to another language translation team.
If possible, please let me know how to upload the korean version image (and how to convert from svg to png).

Many thanks,

**Comment by yoonian** [comment]
I think we should keep the translation version in the same format as the original.

According to https://kubernetes.io/docs/contribute/style/write-new-topic/#adding-images-to-a-topic, the preferred image format is .svg, not .png.

**Comment by jmyung** [comment]
@yoonian 
I totally agree, but we need `ko` directory in `static\images\docs` of website repo.
`svg` and `png` files are not located in the `content/ko/docs`

**Comment by sftim** [comment]
Is there a way that I can help with the logistics of this?

**Comment by jmyung** [comment]
I would re-issue this later, after finding some ways.
